src/main/java/com/jersey/persistence/LoginDao.java

```diff
@@ -1,7 +1,15 @@
 package com.jersey.persistence;
 
 import com.jersey.representations.Login;
+import com.jersey.representations.Person;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.Optional;
 
 public interface LoginDao extends JpaRepository<Login, Long> {
+
+    @Query("SELECT u FROM Login u WHERE LOWER(u.username) = LOWER(:username)")
```

Comment:
why do you need this? 
Cant you create a method in LoginResource. What is the benefit of this @Query.
Do educate me on this matter.